AFQ/api/bundle_dict.py

```diff
@@ -90,186 +77,197 @@ def default18_bd():
-               'prob_map': templates['FA_prob_map'],
-               'start': templates['FA_start'],
-               'end': templates['FA_end']},
-        'FP': {'cross_midline': True,
-               'include': [templates['FP_L'],
-                           callosal_templates['Callosum_midsag'],
-                           templates['FP_R']],
-               'exclude': [],
-               'space': 'template',
-               'prob_map': templates['FP_prob_map'],
-               'start': templates['FP_start'],
-               'end': templates['FP_end']},
-        'pARC_L': {'cross_midline': False,
-                   'include': [templates['SLFt_roi2_L']],
-                   'exclude': [templates['SLF_roi1_L']],
-                   'space': 'template',
-                   'start': templates['pARC_L_start'],
-                   'primary_axis': 2,
-                   'primary_axis_percentage': 40,
-                   'mahal': {
-                       "distance_threshold": 3,
-                       "length_threshold": 5,
-                       "clean_rounds": 10}},
-        'pARC_R': {'cross_midline': False,
-                   'include': [templates['SLFt_roi2_R']],
-                   'exclude': [templates['SLF_roi1_R']],
-                   'space': 'template',
-                   'start': templates['pARC_R_start'],
-                   'primary_axis': 2,
-                   'primary_axis_percentage': 40,
-                   'mahal': {
-                       "distance_threshold": 3,
-                       "length_threshold": 5,
-                       "clean_rounds": 10}},
-        'VOF_L': {'cross_midline': False,
-                  'space': 'template',
-                  'start': templates['VOF_L_start'],
-                  'end': templates['VOF_L_end'],
-                  'inc_addtol': [4, 0],
-                  'primary_axis': 2,
-                  'primary_axis_percentage': 40,
-                  'mahal': {
-                      "distance_threshold": 3,
-                      "length_threshold": 5,
-                      "clean_rounds": 10}},
-        'VOF_R': {'cross_midline': False,
-                  'space': 'template',
-                  'start': templates['VOF_R_start'],
-                  'end': templates['VOF_R_end'],
-                  'inc_addtol': [4, 0],
-                  'primary_axis': 2,
-                  'primary_axis_percentage': 40,
-                  'mahal': {
-                      "distance_threshold": 3,
-                      "length_threshold": 5,
-                      "clean_rounds": 10}}})
+        'Right Anterior Thalamic': {
+            'cross_midline': False,
+            'include': [templates['ATR_roi1_R'],
+                        templates['ATR_roi2_R']],
+            'exclude': [],
+            'space': 'template',
+            'prob_map': templates['ATR_R_prob_map'],
+            'start': templates['ATR_R_start'],
+            'end': templates['ATR_R_end']},
+        'Left Cingulum': {
```

Comment:
We are actually usually only looking at what is called the "Cingulum Cingulate", or also known as the "dorsal cingulum". In older versions we also had something called "cingulum hippocampus", which is the "ventral cingulum". We might want to revisit that one at some point, but it used to be a pretty hard-to-find and noisy bundle. At any rate, this is:
```suggestion
        'Left Cingulum Cingulate': {
```

Reply:
See e.g., https://academic.oup.com/scan/article/11/5/746/1753353